INSTALLER.md

```diff
@@ -91,8 +98,157 @@ For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PAS
+    ```
+7. This value is the tag on the repository. If you would like to use a specific version of Vircadia, typically tags will be named like this: "v2021.1.0-rc"
+    ```
+    Git tag: master
+    # OR, for example
+    Git tag: v2021.1.0-rc
+    ```
+8. This value is the release type. For example, the options are `production`, `pr`, or `dev`. If you are making a build for yourself and others to use then use `production`.
+    ```
+    Release type: DEV
+    # OR, for example we recommend you use
+    Release type: PRODUCTION
+    ```
+9. This value is the release version. Release numbers should be in a format of `YEAR-MAJORVERSION-MINORVERSION` which might look like this: `2021.1.0`.
+    ```
+    Release number: 2021.1.0
+    ```
+10. This value is the build number. We typically use the hash of the most recent commit on that Git tag which might look like this: `fd6973b`.
+    ```
+    Build number: fd6973b
+    ```
+11. This value is the directory that Vircadia will get installed to. You should leave this as the default value unless you are an advanced user.
+    ```
+    Installation dir: /home/ubuntu/Vircadia
+    ```
+12. This value is the number of CPU cores that the Vircadia Builder will use to compile the Vircadia server. By default it will use all cores available on your build server. You should leave this as the default value it gives you for your build server.
+    ```
+    CPU cores to use for Vircadia: 16
+    ```
+13. This value is the number of CPU cores that the Vircadia Builder will use to compile Qt5 (a required component for Vircadia). By default it will use all cores available on your build server. You should leave this as the default value it gives you for your build server.
+    ```
+    CPU cores to use for Qt5: 16
+    ```
+14. It will ask you if you would like to proceed with the specified values. If you're happy with the configuration, type `yes`, otherwise enter `no` and press enter to start over. You can press `Ctrl` + `C` simultaneously on your keyboard to exit.
+15. Vircadia Builder will now run, it may take a while. See this [table](https://github.com/vircadia/vircadia-builder#how-long-does-it-take) for estimated times.
+16. Navigate to the `pkg-scripts` directory.
+    ```
+    cd ../Vircadia/source/pkg-scripts/
+    ```
+17. Generate the .rpm package. Set `RPMVERSION` to the same version you entered for the `Release number` on Vircadia Builder. *Advanced users: the version cannot begin with a letter and cannot include underscores or dashes in it.*
+    ```
+    DEBVERSION="2021.1.0" DEBEMAIL="your-email@somewhere.com" DEBFULLNAME="Your Full Name" ./make-deb-server
+    ```
+18. If successful, the generated .deb package will be in the `pkg-scripts` folder.
+
+##### Amazon Linux 2 | .rpm
+
+1. Ensure you are using an Amazon Linux 2 system, here are the recommended specs:
+    ```
+    AWS EC2 Instance Type: C5a.4xlarge
+    CPU Cores: 16
+    Minimum Disk Space: 40GB
+    ```
```

Comment:
```suggestion
    ```
   You may also use a [virtual machine](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/amazon-linux-2-virtual-machine.html)
```

Keep in mind that you can also use a virtual machine, which is cheaper and probably easier for non-AWS users.
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/amazon-linux-2-virtual-machine.html